src/components/Comments.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+import { CommentsCard } from "./CommentsCard";
+
+export const Comments = ({review}) => {
```

Comment:
(re: my previous comment)

This renders one comment so would be more appropriate to call this one "CommentCard"